region/lowernorfair/east/The Worst Room In The Game.json

```diff
@@ -1685,11 +1778,78 @@
         "and use it to break the bomb blocks below while falling through them."
       ]
     },
+    {
+      "link": [4, 4],
+      "name": "X-Mode Shinecharge, R-Mode Spark Interrupt",
```

Comment:
Probably don't add this.  CF in front of a pirate is close enough to the same thing.